src/main/java/com/google/sps/data/User.java

```diff
@@ -43,4 +50,8 @@ public String getNickname() {
   public boolean getNotifications() {
     return notifications;
   }
+
+  public String getFriendCode() {
```

Comment:
What does a friend code represents?
Could you please add a comment?

Reply:
Done.
The friend codes are the coeds that an owner of a route would need from his friends in order to make them co-owners.